uitools/cpp/Esri/ArcGISRuntime/Toolkit/UtilityNetworkTraceController.cpp

```diff
@@ -572,9 +526,10 @@ UtilityTraceResultListModel* UtilityNetworkTraceController::traceResults()
 
 void UtilityNetworkTraceController::refresh()
 {
+  delete m_selectedUtilityNetwork;
```

Comment:
you need to check this pointer is not null before deleting it

Reply:
It's fine to delete null pointers (resolves into a no-op)